### ISSUE TITLE
Error when trying to display "More info" moderation menu from chat

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#2441]
### Share your bug report, feature request, or comment.

![image](https://user-images.githubusercontent.com/414923/207259823-ee1e51ca-65cd-4514-8792-5d4824c33355.png)

**Comment by dorj222** [comment]
Hi there! I'd like to investigate this issue. I am having a bit hard time replicating this issue. How may I display "More info" moderation menu from chat? :)

**Comment by gabek** [comment]
Good question! You need to be a moderator first. So if you visit the admin `/admin` and go under the chat/users menu, you can select your chat user and give yourself moderator permissions. Then you should be able to hover over chat messages and get the moderation menu. Let me know if you have any other questions!

**Comment by dorj222** [comment]
Thanks a lot for your prompt response! I found it! :)

**Comment by dorj222** [comment]
Hi there! I found two issues that are related to the "More info" moderation menu. The first one was it tried to render/map the connected clients' array without using the unique key. It is currently using the "client.userAgent", which is not always unique and causes the first issue. The second issue was similar to the first one and was caused by rendering/mapping the chat messages table without using the unique key as well. I didn't receive the exact same error message as yours, but I think solving these 2 issues would solve this error message. Should I create a PR for this issue?
